### PR TITLE
Adding egress ability for customer admin.  labeling default namespace with default.

### LINKED ISSUE #1185: Not using network policy plugin?
I think we did in 3.10, perhaps this is a regression from when we upgraded to 3.11?  Re-enable and add e2e test.

### PR DESCRIPTION
**Pull request by kwoodson**
- [x] labeling default project with label to enable networkpolicies
- [x] adding ability to manage egressnetworkpolicies on customer-admin
- [x] need tests to ensure egressnetworkpolicies are working
- [x] need tests for Quotas and LimitRanges 

fixes #1185 

```release-note
Add egressnetworkpolicies to customer admin.  Added tests for limitranges, resourceQuotas, and egressnetworkpolicies.
```

### CODE REVIEW

**Comment by kwoodson** [comment]
Added a `go generate ./...`

**Comment by kwoodson** [comment]
```
[root@master-000000 ~]# oc get project default -o yaml
apiVersion: project.openshift.io/v1
kind: Project
metadata:
  creationTimestamp: 2019-03-01T16:33:11Z
  labels:
    azure.openshift.io/owned-by-sync-pod: "true"
    name: default
  name: default
...
```

**Comment by kwoodson** [comment]
Test is failing with rbac permissions even though permissions have been set:
```
[root@master-000000 ~]# oc get clusterrole customer-admin-project -o yaml
apiVersion: authorization.openshift.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: 2019-03-01T16:37:36Z
  labels:
    azure.openshift.io/owned-by-sync-pod: "true"
  name: customer-admin-project
...
- apiGroups:
  - ""
  attributeRestrictions: null
  resources:
  - egressnetworkpolicies
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch

```

```
• Failure [2.159 seconds]
Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fake]
/home/kwoodson/go/src/github.com/openshift/openshift-azure/test/e2e/specs/customer_admin.go:27
  should be able to manage Quotas and LimitRanges [It]
  /home/kwoodson/go/src/github.com/openshift/openshift-azure/test/e2e/specs/customer_admin.go:115

  Expected error:
      <*errors.StatusError | 0xc000148750>: {
          ErrStatus: {
              TypeMeta: {Kind: "", APIVersion: ""},
              ListMeta: {SelfLink: "", ResourceVersion: "", Continue: ""},
              Status: "Failure",
              Message: "egressnetworkpolicies.network.openshift.io is forbidden: User \"customer-cluster-admin\" cannot create egressnetworkpolicies.network.openshift.io in the namespace \"e2e-test-8tpzk\": no RBAC policy matched",
              Reason: "Forbidden",
              Details: {
                  Name: "",
                  Group: "network.openshift.io",
                  Kind: "egressnetworkpolicies",
                  UID: "",
                  Causes: nil,
                  RetryAfterSeconds: 0,
              },
              Code: 403,
          },
      }
      egressnetworkpolicies.network.openshift.io is forbidden: User "customer-cluster-admin" cannot create egressnetworkpolicies.network.openshift.io in the namespace "e2e-test-8tpzk": no RBAC policy matched
  not to have occurred
```

**Comment by kwoodson** [comment]
Removed changes to the sanitychecker and create the project by the end user.

File: test/e2e/specs/customer_admin.go [hunk 1]
```diff
@@ -104,5 +107,114 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
...
+
+		resQuota := v1.ResourceQuota{
+			Spec: v1.ResourceQuotaSpec{
+				Hard: v1.ResourceList{
+					"configmaps":             resource.MustParse("1"),
+					"persistentvolumeclaims": resource.MustParse("1"),
+				},
+			},
+		}
+		resQuota.ObjectMeta.Name = "testquota"
```
Comment (mjudeikis) [inline test/e2e/specs/customer_admin.go]: nit: you name here `testquota` but below you use abbreviations for limits `testlr`.

**Comment by mjudeikis** [comment]
nit: overall looks ok. Just variable names could be more unified :) either use short everywhere like `lr`, `qa`, or full names as now it's mixed.

File: test/e2e/specs/customer_admin.go (hunk 1 above)
Comment (kwoodson) [reply test/e2e/specs/customer_admin.go]: Made the test names align.

**Comment by jim-minter** [comment]
nice! thanks!
/lgtm

**Pull request merged**